src/test/java/org/openrewrite/java/jackson/UseFormatAlignedObjectMappersTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.jackson;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class UseFormatAlignedObjectMappersTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(new UseFormatAlignedObjectMappers())
+          .parser(JavaParser.fromJavaVersion()
+            .logCompilationWarningsAndErrors(true)
+            .classpathFromResources(new InMemoryExecutionContext(),
+              "jackson-core-2", "jackson-databind-2", "jackson-dataformat-yaml-2", "jackson-dataformat-xml-2"));
+    }
+
+    @DocumentExample
+    @Test
+    void objectMapperDefaultsToJson() {
+        rewriteRun(
+          java(
+            """
+            import com.fasterxml.jackson.databind.ObjectMapper;
+
+            class A {
+                ObjectMapper mapper = new ObjectMapper();
+            }
+            """,
+            """
+            import com.fasterxml.jackson.databind.ObjectMapper;
+            import com.fasterxml.jackson.databind.json.JsonMapper;
+
+            class A {
+                ObjectMapper mapper = new JsonMapper();
```

Comment:
I'm surprised to see this change from a recipe that's documented to handle just the format _wrapping_ constructors. I also wonder if a change like this might surprise folks such that we should be a little more hesitant to make change to just the mandatory cases. Keep in mind that folks will have to review any changes made and this then feels like a partial and perhaps unnecessary change from my quick reading.

I'd be inclined to remove the case for no-args here unless absolutely necessary.

Reply:
No its not, I interpreted it is. will remove.